### ISSUE TITLE
Plugin import from scenario file not working

### ISSUE THREAD

**Issue by amorfinv** [TUDelft-CNS-ATM/bluesky#304]
Hello,

I am getting the following error when trying to load plugins from a scenario file.

![image](https://user-images.githubusercontent.com/78442543/125428287-89668297-3989-4835-8977-b3e59091fa56.png)

The strange thing is that the aircraft appears to follow the waypoints. So it is not really changing the behavior. If I run a `POS` command it tells me that LNAV is switched on (VNAV does not appear to be on)

The M2 plugin subclasses `Route` and `Autopilot` as follows,

```python
"""
M2 Plugin
"""

# Initialize plugin
def init_plugin():

    config = {
        # The name of your plugin
        'plugin_name'      : 'M2',
        'plugin_type'      : 'sim',

        }

    return config


# import the bench players
from bluesky.traffic import Route, Autopilot


class M2Route(Route):
    def __init__(self, acid):
        Route.__init__(self, acid)


class M2Autopilot(Autopilot):
    def __init__(self):
        Autopilot.__init__(self)
```

test.scn has the following commands,

```
00:00:00>PLUGIN LOAD M2
00:00:00>IMPL Route M2Route
00:00:00>IMPL Autopilot M2Autopilot

00:00:00>PAN 48.223775 16.337976
00:00:00>ZOOM 70
00:00:00>CRE D2 M600 48.2210692 16.3285369 22.922002900391682 25 10 
00:00:00>ADDWPT D2 48.221698 16.328936 25 10 
00:00:00>ADDWPT D2 48.2214303 16.3297548 25 30
00:00:00>VNAV D2 ON
00:00:00>LNAV D2 ON
```

After running this scenario, I get the error at the top. However, if I then run the first 3 commands from the scenario in the `bluesky` command line, and then run the rest from a scenario file, there is no issue, see below (VNAV and LNAV are on):

![image](https://user-images.githubusercontent.com/78442543/125429908-5277747a-663e-4280-a153-627d4ca37980.png)

Lastly, if I try to run the scenario from shell `python BlueSky.py --scenfile test.scn` I don't get an error message but a POS command shows that only LNAV is on.

![image](https://user-images.githubusercontent.com/78442543/125429701-389de3a4-e6f0-405b-9ce7-c1e5c78164a9.png)

**Comment by amorfinv** [comment]
I think I figured out the issue. When replacing `Route` with `M2Route` you cannot use `ADDWPT` anymore or any other stack command associated with `Route` since they appear to be hardcoded in basecmds.py. Even if 'Route` is replaced, `ADDWPT` will always go to the `addwptStack()` method in route.py.

The workaround is to just create new stack commands in your plugin that copy the behavior of the base class stack commands. 

A future enhancement should be to make sure base stack commands always go to the selected implementations.

-Andres

**Comment by jooste** [comment]
Fixed in 64fcf9e and 08d94af.
